packages/edge-gateway/src/gateway.js

```diff
@@ -273,7 +273,11 @@ async function cdnResolution(request, env, cache) {
 
   try {
     const res = await pAny(
-      [cache.match(request.url), getFromPermaCache(request, env)],
+      [
+        cache.match(request), // Request from cache API
+        cache.match(request.url), // Request URL from cache API - To be deprecated
```

Comment:
we need to have this for sometime until cache content is migrated. Let's say 24H?